token/program2/src/processor.rs

```diff
@@ -127,8 +127,16 @@ impl Processor {
         program_id: &Pubkey,
         accounts: &[AccountInfo],
         amount: u64,
+        decimals: Option<u8>,
     ) -> ProgramResult {
         let account_info_iter = &mut accounts.iter();
+
+        let mint_account_info = if let Some(decimals) = decimals {
```

Comment:
`decimals.is_some()`?

Reply:
I need the unwrapped decimals below, and am trying to avoid an explicit .unwrap()